### ISSUE TITLE
DrakeVisualizer: Cannot Toggle Hydroelastic Contact Visualizer

### ISSUE THREAD

**Issue by rcory** [RobotLocomotion/drake#16841]
When toggling off hydroelastic contact visualization in Drake Visualizer through the tools menu (`Tools->Hydroelastic Contact Visualizer`), there is no way to toggle it back on. That is, once the hydro visualization is toggled off, it remains off until Drake Visualizer is re-started again.

**Comment by DamrongGuoy** [comment]
I can reproduce the problem as shown in these pictures.

![image](https://user-images.githubusercontent.com/42557859/159591298-c2b178a4-dea9-4d70-91a5-5574ce42d1dc.png)
![image](https://user-images.githubusercontent.com/42557859/159591358-7cdf9bd2-b95c-49c5-8f1c-c9829dc6fcec.png)
![image](https://user-images.githubusercontent.com/42557859/159591489-827f1b1f-0c58-4a5b-81ff-21ec77746868.png)
![image](https://user-images.githubusercontent.com/42557859/159591773-00cad9f4-da37-4121-8a23-75c3ab3ffe44.png)

**Comment by DamrongGuoy** [comment]
I see this error message:
```
Traceback (most recent call last):
  File "/home/damrongguoy/.cache/bazel/_bazel_damrongguoy/aa787bc96edbb97f62958306ba5638ee/external/drake_visualizer/lib/python3.8/site-packages/director/lcmUtils.py", line 119, in handleMessage
    callback(msg)
  File "/home/damrongguoy/.cache/bazel/_bazel_damrongguoy/aa787bc96edbb97f62958306ba5638ee/execroot/drake/bazel-out/k8-opt/bin/tools/drake_visualizer.runfiles/drake/tools/workspace/drake_visualizer/_drake_visualizer_builtin_scripts/show_hydroelastic_contact.py", line 1160, in clear_on_load
    self.visual_model.clear()
  File "/home/damrongguoy/.cache/bazel/_bazel_damrongguoy/aa787bc96edbb97f62958306ba5638ee/execroot/drake/bazel-out/k8-opt/bin/tools/drake_visualizer.runfiles/drake/tools/workspace/drake_visualizer/_drake_visualizer_builtin_scripts/show_hydroelastic_contact.py", line 769, in clear
    body_contact.clear()
  File "/home/damrongguoy/.cache/bazel/_bazel_damrongguoy/aa787bc96edbb97f62958306ba5638ee/execroot/drake/bazel-out/k8-opt/bin/tools/drake_visualizer.runfiles/drake/tools/workspace/drake_visualizer/_drake_visualizer_builtin_scripts/show_hydroelastic_contact.py", line 696, in clear
    om.removeFromObjectModel(self._folder)
  File "/home/damrongguoy/.cache/bazel/_bazel_damrongguoy/aa787bc96edbb97f62958306ba5638ee/external/drake_visualizer/lib/python3.8/site-packages/director/objectmodel.py", line 593, in removeFromObjectModel
    _t.removeFromObjectModel(obj)
  File "/home/damrongguoy/.cache/bazel/_bazel_damrongguoy/aa787bc96edbb97f62958306ba5638ee/external/drake_visualizer/lib/python3.8/site-packages/director/objectmodel.py", line 385, in removeFromObjectModel
    self._removeItemFromObjectModel(self._getItemForObject(obj))
  File "/home/damrongguoy/.cache/bazel/_bazel_damrongguoy/aa787bc96edbb97f62958306ba5638ee/external/drake_visualizer/lib/python3.8/site-packages/director/objectmodel.py", line 257, in _getItemForObject
    return self._objectToItem[obj]
KeyError: <director.objectmodel.ContainerItem object at 0x7fd3c6dc8790>

```

**Comment by DamrongGuoy** [comment]
CC: @rpoyner-tri do you see something obviously wrong?

The ability to toggle hydroelastic visualizer on and off would be very handy.  I hope we can fix it easily.

**Comment by rpoyner-tri** [comment]
It's probably worth a look, but don't forget that drake-visualizer is reaching end-of-life soon. So it's only worth effort until the replacement meshcat solution becomes available.

**Comment by DamrongGuoy** [comment]
I agree.  We should fix it only if it's easy.

I ask for high priority for the Onsite demo next week.

**Comment by rpoyner-tri** [comment]
Had a f2f with @joemasterjohn . I think he will be able to offer some useful advice before too long. I will bow out for  now. :)

**Comment by joemasterjohn** [comment]
One question I had before digging deep into a solution for this is: do you just want to hide all of the hydroelastic contact visualizations, or is there a need to disable the plugin itself? Because if all you need to do is hide the visuals, you can just toggle the eye icon for "Hydroelastic Contact" in the Scene Browser:

![bubble_off](https://user-images.githubusercontent.com/5809905/159771976-c1e54534-1ca7-45cb-a98c-8d6b0525bc47.png)

**Comment by DamrongGuoy** [comment]
That's a very good question, @joemasterjohn.  I let @rcory answer that.

I add that the `eye` works for one snapshot of LCM message.  In the next publish step, the next hydroelastic contact result will show up again.  In other words, the `eye` opens itself for the next message.

**Comment by joemasterjohn** [comment]
Yeah the fact that the `eye` button re-enables for each new LCM message is frustrating.

Regardless, I have a fix for the original problem in #16852. @rcory @DamrongGuoy could you please verify that this fixes the problem for you? Thanks.

**Comment by rcory** [comment]
Verified that the associated PR fixes the issue for me. Thanks!
